### PR TITLE
Refactor color througout interactive mode and dev

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this:**

/kind cleanup

**What does this PR do / why we need it:**

This PR fixes the following:
- Updates the interactive mode so it's better looking when outputting
  configuration so it's similar to the rest of the output:

```sh
? Is this correct? Yes
 ✓  Downloading devfile "nodejs" from registry "DefaultDevfileRegistry" [1s]

↪ Container Configuration "runtime":
  OPEN PORTS:
    - 3000
  ENVIRONMENT VARIABLES:

? Select container for which you want to change configuration? runtime
? What configuration do you want change? Delete port "3000"
? What configuration do you want change? NOTHING - configuration is correct

↪ Container Configuration "runtime":
  OPEN PORTS:
  ENVIRONMENT VARIABLES:

? Select container for which you want to change configuration?  [Use arrows to move, type to filter]
  runtime
> NONE - configuration is correct
```

- Removes issue of fgGreen not appearing correctly because of direct use
  of `color` instead of log/status.go. This is due to color not
  detecting if the terminal supports the color, which is done in
  `log/status.go` instead.

**Which issue(s) this PR fixes:**

Fixes https://github.com/redhat-developer/odo/issues/5843

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: pkg/init/backend/interactive.go [hunk 1]
```diff
@@ -220,18 +218,24 @@ func PrintConfiguration(config asker.DevfileConfiguration) {
...
+
 		for key, value := range container.Envs {
-			color.New(color.Bold, color.FgWhite).Printf("   - %s = %s\n", key, value)
+			fmt.Printf("    - %s = %s\n", key, value)
 		}
+
 	}
+
+	// Make sure we add a newline at the end
+	fmt.Println()
```
Comment (dharmit) [inline pkg/init/backend/interactive.go:238]: Please don't use `fmt.Printf` or `fmt.Println` in the business layer. It should ideally be used only in CLI layer. Use `fmt.Fprintf` with stdout as `io.Writer`.

**Review by dharmit — CHANGES_REQUESTED** [review]

**Comment by cdrage** [comment]
Thanks! Updated @dharmit

**Review by rm3l — COMMENTED** [review]
/approve

**Comment by cdrage** [comment]
Re-added

/lgtm

Because I forgot to sign the commit

**Comment by rm3l** [comment]
/override ci/prow/v4.10-integration-e2e

Tests passing in IBM Cloud.

**Pull request merged**